Doc/library/weakref.rst

```diff
@@ -63,9 +63,9 @@ or :class:`finalize` is all they need -- it's not usually necessary to
 create your own weak references directly.  The low-level machinery is
 exposed by the :mod:`weakref` module for the benefit of advanced uses.
 
-Not all objects can be weakly referenced; those objects which can include class
-instances, functions written in Python (but not in C), instance methods, sets,
-frozensets, some :term:`file objects <file object>`, :term:`generators <generator>`,
+Not all objects can be weakly referenced. Objects which support weak references
```

Comment:
Perhaps change "which" to "that".

Reply:
I agree with your suggestion. However, is it worth to open a new PR for fixing this?